concordium-consensus/src/Concordium/Queries.hs

```diff
@@ -664,6 +676,53 @@ getPoolStatus blockHashInput mbid = do
 getRegisteredBakers :: forall gsconf finconf. BlockHashInput -> MVR gsconf finconf (BlockHash, Maybe [BakerId])
 getRegisteredBakers = liftSkovQueryBHI (BS.getActiveBakers <=< blockState)
 
+-- |Get a list of delegators for a given pool at the end of a given block.
+getPoolDelegators :: forall gsconf finconf. BlockHashInput -> Maybe BakerId -> MVR gsconf finconf (BlockHash, Maybe (Maybe [PoolDelegatorInfo]))
```

Comment:
Please document the return type.

Reply:
I have added a bit for the documentation and defined a proper error type here.